0000-template.md

```diff
@@ -21,18 +21,17 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 Why are we doing this? What use cases does it support? What is the expected outcome?
 
-#Guide-level-explanation
+# Guide-level explanation
 [guide-level-explanation]: #guide-level-explanation
 
-Explain the proposal as if it were already included in the language, or in the kernel, and you were teaching it to another CC+ programmer or Cpcdos OS2.2 contributor. This usually means :
+Explain the proposal as if it was already included in the language, or in the kernel, and you was teaching it to another CC+ programmer or Cpcdos OS2.2 contributor. This usually means :
 
 - Introducing new named concepts.
 - Explaining the functionality largely through examples.
 - Explaining how CC+ programmers should *think* about the feature and how it should impact on the way they use it. The impact should be explained in as concrete a way as possible.
-- Explain how the feature fits into the kernel and its relationship to different parts of it.
+- Explaining how the feature fits into use and the relationship between its different parts.
 - If applicable, provide examples of error messages, depreciation warnings or migration advice.
-- If applicable, describe the differences between teaching this functionality to existing CC+ programmers and to new CC+ programmers. The same applies to features to be added to the kernel.
-
+- If applicable, describe the differences between teaching this functionality to existing CC+ programmers and to new CC+ programmers.
```

Comment:
I completely forgot on the previous commit to ask you, what does this sentence means.

Why should you make a difference between new programmer to more experienced one ?

Reply:
The new programmer needs to learn how the kernel and/or the language works. And they may make more mistakes than an experienced programmer who knows how to avoid them. I think I should have added the fact that it is essential to ask an experienced programmer about the project if you are a beginner and don't know where to go.

Reply:
Yes but why should it be on a RFC ?

Reply:
Sorry, but I can't see any clearer :

The old dev will have to learn new things each time a new feature is added, while the new dev will have to learn all the features that were added before he arrived. I don't see how I can make it any clearer.

Reply:
Yes but there is no reason to do that